lib/tapioca/compilers/todos_compiler.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+# typed: strong
+
+module Tapioca
+  module Compilers
+    # Taken from https://github.com/sorbet/sorbet/blob/master/gems/sorbet/lib/todo-rbi.rb
+    class TodosCompiler
+      extend(T::Sig)
+
+      sig do
+        returns(String)
+      end
+      def compile
+        rbi = String.new
+        list_todos.each_line do |line|
+          next if line.include?("<") || line.include?("class_of")
+          rbi << "module #{line.strip.gsub('T.untyped::', '')}; end\n"
+        end
+        rbi
```

Comment:
Not that it matters a lot, but I am a fan of collecting and joining, whenever possible (no-side effect functional programming tendency). Also I think joining an array of strings would be slightly faster and more efficient than appending to a string. So I would suggest:
```suggestion
        list_todos.each_line.map do |line|
          next if line.include?("<") || line.include?("class_of")
          module_name = line.strip.gsub('T.untyped::', '')
          "module #{module_name}; end"
        end.compact.join("\n")
```

Reply:
It's not entirely equivalent as the `join` strips the trailing `\n` hence the addition here: https://github.com/Shopify/tapioca/pull/51/commits/574529513d4bf3af5bd9437958d19e828694c298#diff-595c19bc567be3296e5404c802a7bc1fR67